src/controllers/mainController.ts

```diff
@@ -443,7 +443,11 @@ export default class MainController implements vscode.Disposable {
 
 		// Generate Azure Function command
 		this._context.subscriptions.push(vscode.commands.registerCommand(Constants.cmdCreateAzureFunction, async (node: TreeNodeInfo) => {
-			const connectionDetails = ConnectionCredentials.createConnectionDetails(node.connectionInfo);
+			let connectionInfo = node.connectionInfo;
+			// set the database containing the selected table so it can be used
+			// for the initial catalog property of the connection string
+			connectionInfo.database = node.parentNode.parentNode.label as string;
```

Comment:
You shouldn't use the label - that may have additional text added to it. e.g. for external tables : 

![image](https://user-images.githubusercontent.com/28519865/156411209-ec0dc172-241f-4c03-9f9d-70be8f921ada.png)

Use node.metadata.name instead.